src/Shared/INodePacketTranslator.cs

```diff
@@ -8,11 +8,7 @@
 using System;
 using System.Collections.Generic;
 using System.Globalization;
-using System.Text;
 using System.IO;
-using Microsoft.Build.Collections;
-using Microsoft.Build.Execution;
-using Microsoft.Build.Shared;
```

Comment:
We've mostly been avoiding `using` cleanups while we're in two branches, but no need to undo this . . .